### DISCUSSION TITLE
Settings in mobile app

### DISCUSSION THREAD

**Discussion by romw314** [community/community#64481] [Mobile]
### Select Topic Area

Product Feedback

### Body

I want to change repository settings in the mobile app, but I can't find them.
Can someone add the setting editing to the app.

Thanks.

**Comment by arafajs** [comment]
Unfortunately, changing repository settings in the GitHub mobile app is impossible. You can only change repository settings on the GitHub website.
You can open the browser on your mobile. and log in to your GitHub account.

**Reply by romw314** [reply]
yes! and always switch apps between Edge and microsoft authenticator when logging in.

**Reply by RaistlinD2x** [reply]
Why would that not be a feature?

**Comment by Salis09** [comment]
What is java

**Reply by romw314** [reply]
Why you're asking this question in my topic?
Instead of this, create a new topic.
Java is a programming language, for me.

**Reply by Dvcutie04** [reply]
> Why you're asking this question in my topic?
> 
> Instead of this, create a new topic.
> 
> Java is a programming language, for me.
> Do I need your permission for in app notifications?

**Comment by Dvcutie04** [comment]
My email is dvccobbold04@gmail.com if you need to reach me directly

**Reply by Dvcutie04** [reply]
Do I need your permission for in app notifications and modifications?

**Reply by Ceryozha** [reply]
Потому что нету перевода

**Reply by Ceryozha** [reply]
Никто не понимает что куда нету поддержки вы не поддерживаете нас мы не поддерживаем вас вот так не выходит вот так нас всех поссорили чтобы мы были между собой как звери

**Comment by Ceryozha** [comment]
Please make an application with Russian or Ukrainian language, please!!!!!!!!!!!!
Is it really that difficult for you? Please, there is a war in our country in Ukraine. I really ask for help. We simply need an application with the Ukrainian language.

And also let us know that you have added our language to the application, thank you, please

**Reply by Basill69** [reply]
I agree with the author of this request!We need it in Ukraine.

**Reply by Ceryozha** [reply]
No, the Github guys are weak, they can't handle this, they don't know how to do that.

**Comment by Se7enty11** [comment]
Потрібна Українська мова в мобільному додатку GitHub, так як дуже багато користувачів використовували саме цей додаток так як він дуже корисний.

**Comment by Ceryozha** [comment]
No, the Github guys are weak, they can't handle this, they don't know how to do that.

**Comment by Ceryozha** [comment]
Github developers aren't trained for this kind of thing.
